puppet/modules/slave/manifests/rbenv.pp

```diff
@@ -0,0 +1,24 @@
+# rbenv Configuration
+# @api private
+class slave::rbenv {
+  class { 'rbenv':
+    #owner       => 'jenkins',
+    #group       => 'jenkins',
+    #install_dir => '/home/jenkins/.rbenv',
+    #require     => User['jenkins'],
+  }
+
+  ensure_packages(['perl'])
+
+  rbenv::plugin { 'rbenv/ruby-build':
+  }
+  rbenv::build { '3.1.0':
+    env => ['RUBY_CONFIGURE_OPTS=--without-openssl --with-openssl-dir=/usr'],
```

Comment:
This sounds conflicting: without openssl, but with an openssl option.

Reply:
I believe it is saying "do not build openssl yourself and use it from over there instead"